### ISSUE TITLE
[Testing] Fix axe-core a11y tests

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3493]
### Provide a general summary of the issue here

A11y tests for components right now are using only rules from WCAG 2.2 levels A and AA. This is not sufficient. 
Rules from WCAG 2.0 and WCAG 2.1 levels A and AA should be added to each a11y test. 

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
